src/Indicator.vala

```diff
@@ -19,6 +19,7 @@
 
 public class Power.Indicator : Wingpanel.Indicator {
     private const double LOW_BATTERY_PERCENTAGE = 20;
+    private bool is_desktop { get; set; default = false; }
```

Comment:
```suggestion
    private bool is_desktop = false;
```

I noticed this just now; maybe we don't need `get` and `set` because this is not a property